main/src/ui/chat_input/chat_input_controller.vala

```diff
@@ -68,7 +88,20 @@ public class ChatInputController : Object {
         input_field_status = status;
 
         chat_input.set_input_state(status.message_type);
-        status_description_label.label = status.message;
+        if (status.contains_markup){
+            status_description_label.set_markup("%s".printf(Markup.escape_text(status.message)));
+            status_description_label.label = status.message;
+            status_description_label.activate_link.connect(() => {
```

Comment:
Same as above, you are connecting to this signal every time `set_input_field_status()` is invoked instead of only one time.
Also you should verify that the link is really intended to open the conversation details, by checking that the link uri (first lambda parameter) equals the one specified above ("x-dino:open-conversation-details").